File: jna/src/main/java/dark/boxabs.java
package dark;

import com.sun.jna.Pointer;
import com.sun.jna.Structure;

import java.util.Arrays;
import java.util.List;

public class boxabs extends Structure {
    public float left,right,top,bot;

    public boxabs() {
        super();
    }

    public boxabs(Pointer peer) {
        super(peer);
    }

    @Override
    protected List<String> getFieldOrder() {
        return Arrays.asList("left","right","top","bot");
    }
}

File: jna/src/main/java/dark_kotlin/boxabs.kt
package com.sl.microservice.ai.runtime.impl.darknet.dark

import com.sun.jna.Pointer
import com.sun.jna.Structure

class boxabs : Structure {
    val left = 0f
    val right = 0f
    val top = 0f
    val bot = 0f

    constructor() : super()

    constructor(peer: Pointer) : super(peer)

    override fun getFieldOrder(): List<String> {
        return listOf("left", "right", "top", "bot")
    }
}
